docs/enhancement-proposals/off-ep-0005.md

```diff
@@ -0,0 +1,134 @@
+# OFF-EP 0005 — Allow different electrostatics methods to be used on period and non-periodic systems
+
+**Status:** Proposed
+
+**Authors:** Matt Thompson and John Chodera
+
+**Stakeholders:** Simon Boothroyd, Jeffrey Wagner, David Mobley, John Chodera
+
+**Acceptance criteria:** Unanimity
+
+**Created:** 2020-03-15
+
+**Discussion:** [Issue #29](https://github.com/openforcefield/standards/issues/29)
+
+**Implementation:** [``openff-standards``](https://github.com/openforcefield/openff-standards)
+
+## Abstract
+
+This change refines the way the `<Electrostatics>` tag defines behavior in periodic
+(i.e., condensed-phase) and non-periodic (i.e., gas phase) systems to clarify the
+intended true electrostatics models in each case. Implementations are permitted
+to make approximations to these specified models---e.g. Particle-Mesh Ewald (PME)---
+as a controlled approximation to Ewald) provided the approximation accuracy is controlled.
+
+## Motivation and Scope
+
+In version 0.3 of the `<Electrostatics>` tag, the default electrostatics method is
+`method="PME"`, with `reaction-field` also a permitted choice.
+
+These definitions present several issues that this OFF-EP attempts to solve:
+* `PME` is intended to be a permissible approximation to the true electrostatics model, Ewald
+* The boundary conditions (e.g. dielectric at infinity) for the Ewald sum are not specified
+* The Ewald method is only intended for periodic systems; unmodified vacuum electrostatics are intended for non-periodic systems
+* The treatment of intramolecular electrostatics exceptions is unspecified
+* The choice `reaction-field` does not uniquely specify the functional form for the true reaction-field model intended; many variants are available
+* The solvent dielectric constant was not specified.
+* Cutoffs were not specified
+* The physical constants used to compute the potential were unspecified
+
+To solve these issues, this OFF-EP proposes:
+* The `method` attribute is replaced with `periodic_potential` in analogy to other parameters that use the `potential` term to specify the functional form or a common choice
+* The `periodic_potential` attribute defaults to `Ewald3D-ConductingBoundary` as a valid keyword that states that the Ewald periodic sum with conducting boundary conditions should be the true potential used for periodic systems
+* PME (and other methods) are permissible approximations to Ewald as long as they are controlled.
+* For reaction field or other methods, the `periodic_potential` can specify the exact functional form used for the periodic potential or a keyword denoting a common choice, along with the optional `cutoff` and `solvent_dielectric` attributes
```

Comment:
(not blocking) We currently also specify `solvent_dielectric` in the [GBSA section](https://github.com/openforcefield/standards/blob/17e57c934e449b8ab79462bd2db245d73b26913b/docs/standards/smirnoff.md#gbsa). As far as I understand the math, it would be fine for these to have a different value, but I wanted to double check that nobody else sees any fundamental problem that this could create.

Reply:
I think our philosophy is to keep constants like this local to their parameter sections as much as possible.

However, this does suggest we may want to group multiple tags hierarchically in the future if there is a desire to ensure information is shared.